ui/components/app/snaps/snap-ui-renderer/index.scss

```diff
@@ -15,6 +15,14 @@
     height: 100%;
   }
 
+  &__panel > *:first-child {
```

Comment:
Why is this needed now?

Reply:
Won't this rule also match nested panels? So nested panels first and last element will also have no margins?

Reply:
The existing code is meant to do this already too? 🤔  https://github.com/MetaMask/metamask-extension/blob/7420ea9a5dca41dd2b74c9d19c88045c9588d8fd/ui/components/app/snaps/snap-ui-renderer/index.scss#L30-L40